frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -801,9 +801,14 @@ export const funnelLogic = kea<funnelLogicType<openPersonsModelProps>>({
                             ...getBreakdownStepValues(baseStep, 0, true),
                             isBaseline: true,
                             breakdownIndex: 0,
-                            steps: steps.map((s) =>
-                                Object.assign({}, s, { nested_breakdown: undefined, breakdown_value: 'Baseline' })
-                            ),
+                            steps: steps.map((s) => {
+                                return Object.assign({}, s, {
```

Comment:
nit: `=> { return Object.assign({}, s, {` can be shortened to `({ ...s, {` with corresponding closing tags.

Reply:
Overall I don't quite get why and how this is synthesized in the logic instead of "Baseline" coming from the backend. 🤔 Can someone lay this out for me?

Reply:
Your intuitions spot on, ideally we wouldn't want to create a baseline step in the frontend here... just one of the many pieces of tech debt in funnelLogic.

Today an extra baseline step is prepended to the flattened list of breakdown steps so that this a baseline key shows up in the table legend.